config/locales/en/documents/show.yml

```diff
@@ -83,3 +87,6 @@ en:
         topic_update_error:
           title: Something has gone wrong
           description_govspeak: Something went wrong when saving your changes. Please try again or [raise a support request](https://support.publishing.service.gov.uk/technical_fault_report/new).
+        unwithdraw_error:
+          title: Something has gone wrong
+          description_govspeak: Something went wrong when trying to publish the withdrawn document. Please try again or [raise a support request](https://support.publishing.service.gov.uk/technical_fault_report/new).
```

Comment:
should this be "republish the withdrawn document" ? as per the republish terminology used in line 15

Reply:
yep nice spot